org.nickvision.money.gnome/src/views/mainwindow.cpp

```diff
@@ -36,6 +36,10 @@ namespace Nickvision::Money::GNOME::Views
         m_currencyConverterPage{ GTK_WINDOW(m_window) }
     {
         //Setup Window
+        if(m_window == nullptr)
```

Comment:
There's no need for this if statement since the `fromBlueprint` function will throw an error if the blueprint files didn't get compiled.